docs-website/package.json

```diff
@@ -17,7 +17,7 @@
     "generate": "rm -rf genDocs genStatic && mkdir genDocs genStatic && yarn _generate-docs && mv docs/* genDocs/ && rmdir docs",
     "generate-rsync": "mkdir -p genDocs genStatic && yarn _generate-docs && rsync -v --checksum -r -h -i --delete docs/ genDocs && rm -rf docs",
     "lint": "prettier -w generateDocsDir.ts sidebars.js src/pages/index.js",
-    "lint-check": "prettier -l generateDocsDir.ts sidebars.js src/pages/index.js",
+    "lint-check": "prettier -l generateDocsDir.ts sidebars.js src/pages/index.js && find ../docs -name '*.md' -exec markdown-link-check -q {} -c markdown-link-check-config.json \\;",
```

Comment:
we have docs in places other than `../docs`

might be easier to check over `./genDocs`